### DISCUSSION TITLE
Adding Tiburonians to the Races

### DISCUSSION THREAD

**Discussion by FortunaeSD** [bcholmes/StarTrek2d20#220] [Ideas]
Could you please add the Canon species in Continuing Voyages to the character creation?  In particular the Tiburoneans?  Thanks in advance.

**Comment by Felderburg** [comment]
For reference, Continuing Mission does have a writeup: https://continuingmissionsta.com/2018/03/14/tiburonian/

**Comment by bcholmes** [comment]
I think this was added a while ago.

<img width="1059" alt="Screenshot 2025-01-11 at 11 32 59 AM" src="https://github.com/user-attachments/assets/df336e0d-b921-4c16-a7c2-e127a82025b5" />
